ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/orbspecific/javaidl/recoverycoordinators/JavaIdlRCServiceInit.java

```diff
@@ -248,7 +248,7 @@ private static String initRCKey() {
     }
 
     private static final String POA_NAME_PREFIX = "RcvCo-";
-    private static String OBJ_KEY_PREFIX = ""; // is there an equivalent of JacORBs jacorb.implname
```

Comment:
Same comment as above, this might be useful even after we stop supporting use of JacORB?